<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyBoard</class>
 <widget class="QWidget" name="keyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="keyButton" name="Q">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="keyButton" name="W">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="keyButton" name="E">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="keyButton" name="R">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="keyButton" name="T">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="keyButton" name="Y">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="keyButton" name="U">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="keyButton" name="I">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="keyButton" name="O">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="keyButton" name="P">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="keyButton" name="A">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="keyButton" name="S">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="keyButton" name="D">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="keyButton" name="F">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="keyButton" name="G">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="keyButton" name="H">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="keyButton" name="J">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="keyButton" name="K">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="keyButton" name="L">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="keyButton" name="Z">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="keyButton" name="X">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="keyButton" name="C">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="keyButton" name="V">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="keyButton" name="B">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="keyButton" name="N">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="keyButton" name="M">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="keyButton" name="lock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>unlocked</string>
   </property>
  </widget>
  <widget class="keyButton" name="num1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="keyButton" name="num2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="keyButton" name="num3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="keyButton" name="num4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="keyButton" name="num5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="keyButton" name="num6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="keyButton" name="num7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="keyButton" name="num8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="keyButton" name="num9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="keyButton" name="num0">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="keyButton" name="backspace">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="keyButton" name="point">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="keyButton" name="leftLine">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="keyButton" name="rightLine">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>\</string>
   </property>
  </widget>
  <widget class="keyButton" name="semicolon">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>;</string>
   </property>
  </widget>
  <widget class="keyButton" name="question">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="keyButton" name="minus">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="keyButton" name="comma">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>keyButton</class>
   <extends>QPushButton</extends>
   <header>keybutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>num1</tabstop>
  <tabstop>W</tabstop>
  <tabstop>E</tabstop>
  <tabstop>R</tabstop>
  <tabstop>T</tabstop>
  <tabstop>Y</tabstop>
  <tabstop>U</tabstop>
  <tabstop>I</tabstop>
  <tabstop>O</tabstop>
  <tabstop>P</tabstop>
  <tabstop>A</tabstop>
  <tabstop>S</tabstop>
  <tabstop>D</tabstop>
  <tabstop>F</tabstop>
  <tabstop>G</tabstop>
  <tabstop>H</tabstop>
  <tabstop>J</tabstop>
  <tabstop>K</tabstop>
  <tabstop>L</tabstop>
  <tabstop>Z</tabstop>
  <tabstop>X</tabstop>
  <tabstop>C</tabstop>
  <tabstop>V</tabstop>
  <tabstop>B</tabstop>
  <tabstop>N</tabstop>
  <tabstop>M</tabstop>
  <tabstop>lock</tabstop>
  <tabstop>Q</tabstop>
  <tabstop>num2</tabstop>
  <tabstop>num3</tabstop>
  <tabstop>num4</tabstop>
  <tabstop>num5</tabstop>
  <tabstop>num6</tabstop>
  <tabstop>num7</tabstop>
  <tabstop>num8</tabstop>
  <tabstop>num9</tabstop>
  <tabstop>num0</tabstop>
  <tabstop>backspace</tabstop>
  <tabstop>point</tabstop>
  <tabstop>leftLine</tabstop>
  <tabstop>rightLine</tabstop>
  <tabstop>semicolon</tabstop>
  <tabstop>question</tabstop>
  <tabstop>minus</tabstop>
  <tabstop>comma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
